token/js/src/state/account.ts

```diff
@@ -82,23 +82,11 @@ export const AccountLayout = struct<RawAccount>([
 /** Byte length of a token account */
 export const ACCOUNT_SIZE = AccountLayout.span;
 
-/**
- * Retrieve information about a token account
- *
- * @param connection Connection to use
- * @param address    Token account
- * @param commitment Desired level of commitment for querying the state
- * @param programId  SPL Token program account
- *
- * @return Token account information
- */
-export async function getAccount(
-    connection: Connection,
+function info2Account(
```

Comment:
nit: can we call this `unpackTokenAccount` or `fromAccountInfo`? The `2` is unfortunately not in our conventions